src/components/ScriptOptionBlock.tsx

```diff
@@ -105,6 +121,12 @@ export const ScriptOptionBlock: React.FC<ScriptOptionBlockProps> = (props) => {
       {emptyScript && (
         <div className={classes.errorLabel}>Script cannot be empty</div>
       )}
+      {invalidCampaignVariablesUsed.length > 0 && (
+        <div className={classes.errorLabel}>
+          Script cannot use a campaign variable without a variable:{" "}
```

Comment:
Nitpick: I think this is probably what was meant?
```suggestion
          Script cannot use a campaign variable without a value:{" "}
```